lib/livebook/notebook/learn/github_stars.livemd

```diff
@@ -141,7 +141,16 @@ Execute the cell below. You should see a list of GitHub users who starred the re
 
 ```elixir
 repo_name = "livebook-dev/livebook"
-{:ok, stargazers} = GitHubApi.stargazers(repo_name)
+
+stargazers =
+  case GitHubApi.stargazers(repo_name) do
+    {:ok, stargazers} ->
+      stargazers
+
+    {:error, error_message} ->
+      IO.puts(error_message)
+      []
```

Comment:
What about this?

```suggestion
      Kino.interrupt!(:error, error_message)
```

Since continuing with an empty list won't produce anything interesting.

Reply:
Ah, this is a test run, I see!